internal/pkg/service/buffer/storage/naming.go

```diff
@@ -0,0 +1,18 @@
+package storage
+
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/compression"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+func SliceFilename(ft FileType, ct compression.Type) (string, error) {
+	filename := sliceFilename
+	switch ft {
+	case FileTypeCSV:
+		filename += ".csv"
+	default:
+		return "", errors.Errorf(`unexpected file type "%s"`, ft)
+	}
+
+	return compression.Filename(filename, ct)
+}
```

Comment:
Tady asi počítáš jednu slice per worker takže nepočítáš s nějakým uniq nebo counterem, že? Protože když ten slice pak nahraješ na stage tak jich tam asi može byt víc a všecky sa nemožů menovat slice.csv

Reply:
jj v dalsich PRs sa to este vyvija :) 

Je tam `slice.LocalStorage.Filename` (plnena z tehto `SliceFilename` funkcie).

A potom tam je `slice.StagingStorage.Path` ... tam sa prida ako prefix `VolumeID` (tj. unikatna hodnota per pod).

![image](https://github.com/keboola/keboola-as-code/assets/19371734/3bd9da29-4778-4735-bf65-6f06b55e5645)